README.md

```diff
@@ -62,14 +62,11 @@ Imports should be grouped into the sections below and in sorted order.
 
 See LICENSE.txt for license information
 
-## Lead
 
-+ **Edwin Skidmore <edwin@iplantcollaborative.org>**
-
-## Authors 
+## Original Authors 
```

Comment:
I think this file was modified by lucas and I am somehow picking up his changes.  This file is not needed for this pull request and I am not certain how to get rid of it.